src/containers/LoginForm.js

```diff
@@ -1,12 +1,38 @@
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
+import './LoginForm.css'
 
 class LoginForm extends Component {
+  constructor() {
+    super();
+    this.state = {
+      email: '',
+      password: ''
+    }
+  }
+
+  handleInputChange(event) {
+    this.setState({ [event.target.name]: event.target.value })
+  }
+
   render() {
     return (
-      <div>
-        <h1>LoginForm</h1>
-      </div>
+      <section className="login-section">
+        <div className="form-container">
+          <h1>Login Form</h1>
+          <form>
+            <div className="input-container">
+              <label for="email-input">Email:</label>
+              <input id="email-input" type="text" placeholder="Email Address" value={this.state.email} name="email" onChange={event => this.handleInputChange(event)} />
+            </div>
+            <div className="input-container">
+              <label for="password-input">Password:</label>
+              <input id="password-input" type="text" placeholder="Password" value={this.state.password} name="password" onChange={event => this.handleInputChange(event)} />
+            </div>
+            <button className="submit-btn">Submit</button>
+          </form>
+        </div>
+      </section>
     );
   }
 }
```

Comment:
Just making a note to be sure we come back and add propTypes for this LoginForm componenet.

Reply:
Absolutely. Likewise for the NavBar component (depending on how we build our logout functionality) as well as eventually for the MovieShowPage component once we've fleshed out the props there.